main.cpp

```diff
@@ -48,20 +48,33 @@ Triangles triangles_from_scene(const aiScene* scene) {
 
             for (aiFace face : make_range(mesh.mFaces, mesh.mNumFaces)) {
                 assert(face.mNumIndices == 3);
-                triangles.push_back(
-                    Triangle{// vertices
-                             {{T * mesh.mVertices[face.mIndices[0]],
-                               T * mesh.mVertices[face.mIndices[1]],
-                               T * mesh.mVertices[face.mIndices[2]]}},
-                             // normals
-                             {{Tp * mesh.mNormals[face.mIndices[0]],
-                               Tp * mesh.mNormals[face.mIndices[1]],
-                               Tp * mesh.mNormals[face.mIndices[2]]}},
-                             ambient,
-                             diffuse,
-                             emissive,
-                             reflective,
-                             reflectivity});
+
```

Comment:
Hm, I wonder if we should move the complete `triangles_from_scene` function at some point. It's good for now.